hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -630,13 +630,15 @@ private void handleLogLevelReset(HttpPostCommand command) throws UnsupportedEnco
 
     protected void handleConfigReload(HttpPostCommand command) throws UnsupportedEncodingException {
         prepareResponse(
+                SC_500,
```

Comment:
500 is not correct HTTP code for not licensed, but it's what we return for other EE endpoints like WAN replication. We used to return 200 with an error message here